demo/regl.js

```diff
@@ -0,0 +1,373 @@
+            (2.0 * yR * zR) + (2.0 * wR * xR),
+            0,
+            (2.0 * xR * zR) + (2.0 * wR * yR),
+            (2.0 * yR * zR) - (2.0 * wR * xR),
+            1.0 - (2.0 * xR * xR) - (2.0 * yR * yR),
+            0,
+            t0,
+            t1,
+            t2,
+            1
+            );
+
+      vec4 leftWorldSpace = weightedMatrix * vec4(aVertexPosition, 1.0);
+      float y = leftWorldSpace.z;
+      float z = -leftWorldSpace.y;
+      leftWorldSpace.y = y;
+      leftWorldSpace.z = z;
+
+      
+      vTextureCoord = aTextureCoord;
+    
+
+      gl_Position = uPMatrix * uMVMatrix * leftWorldSpace;
+     }`,
+
+      frag: `
+        precision mediump float;
+        varying vec2 vTextureCoord;
+        varying vec3 vLightWeighting;
+
+        uniform sampler2D uSampler;
+    
+
+    void main (void) {
+      vec4 textureColor = texture2D(uSampler, vec2(vTextureCoord.s, vTextureCoord.t));
+      gl_FragColor = vec4(textureColor.rgb, textureColor.a);
+    }`,
+
+      attributes: {
+        aVertexPosition: {
+          buffer: regl.buffer({
+            data: cowboy.vertexPositions,
+            type: 'float32',
+          }),
+          size: 3
+        },
+        
+        aVertexNormal: {
+          buffer: regl.buffer({
+            data: vertexData.vertexNormals,
+            type: 'float32'
+          }),
+          size: 3
+        },
+        
+        aJointIndex: {
+          buffer: regl.buffer({
+            data: vertexData.vertexJointAffectors,
+            type: 'float32'
+          }),
+          size: 4
+        },
+        
+        aJointWeight: {
+          buffer: regl.buffer({
+            data: vertexData.vertexJointWeights,
+            type: 'float32'
+          }),
+          size: 4
+        },
+
+        aTextureCoord: {
+          buffer: regl.buffer({
+            data: vertexData.vertexUVs,
+            type: 'float32'
+          }),
+        }
+      },
+
+      primitive: 'triangles',
+      count: vertexData.vertexPositionIndices.length,
+      
+      elements: regl.elements({
+        data: vertexData.vertexPositionIndices,
+        type: 'uint16',
+        primitive: 'triangles'
+      }),
+      
+      uniforms: Object.assign({
+        boneRotQuaternions: regl.prop('boneRotQuaternions'),
+        boneTransQuaternions: regl.prop('boneTransQuaternions'),
+        uMVMatrix: modelMatrix,
+        uPMatrix: drawOpts.perspective,
+        uSampler: regl.texture(assets.texture),
+      }, new Uint32Array(18).reduce((accum, value, index) => {
+        accum['boneRotQuaternions['+index+']'] = regl.prop('boneRotQuaternions['+index+']');
+        return accum;
+      }, {}), new Uint32Array(18).reduce((accum, value, index) => {
+        accum['boneTransQuaternions['+index+']'] = regl.prop('boneTransQuaternions['+index+']');
+        return accum;
+      }, {})),
+    });
+
+    var currentTime = 0;
+    regl.frame(({time}) => {
+      currentTime += time;
```

Comment:
Looks like it's moving through the keyframes so quickly that it appears to jitter.

In the original demo we divide the time delta by `1000`

ex: https://github.com/chinedufn/skeletal-animation-system/blob/master/demo/demo.js#L109

So hopefully something like that can help with the jitter